scripts/dockerbot/dockerbot.py

```diff
@@ -176,6 +177,17 @@ def cache_files(self):
             if self.requirements_update:
                 self.args['update_pip'] = True
 
+    def get_base_branch(self, pr):
+        token = self.args['gh_token']
+        owner = self.args['gh_owner']
+        repo = self.args['gh_repo']
+        if token:
```

Comment:
Yeh some of this code is gnarly and copied from another bit......
```self.base_branch = self.get_base_branch(self.args['pr']) or self.base_branch```
probably wants to be
```self.base_branch =  self.base_branch or self.get_base_branch(self.args['pr'])```